examples/kurth/input_kurth.in

```diff
@@ -35,3 +32,6 @@ constf1.kt = 1.0
 ###############################################################################
 algo.particle_shape = 2
 algo.space_charge = false
+
+amr.n_cell = 40 40 32
+geometry.prob_relative = 1.0
```

Comment:
The above test, "input_kurth.in", uses only a Kurth distribution and a constant focusing element (without space charge), features that are already tested elsewhere.  I suggest that this test can be removed.  The two important Kurth tests are "input_kurth_periodic.in" (without space charge) and "input_kurth_10nC_periodic.in" (with space charge).

Reply:
I agree - thanks for the update, I will add one more push that removes `input_kurth.in` :)

What's the difference between `input_kurth_10nC_periodic.in` and `input_kurth_10nC.in` (both w/ space charge)? We keep both? :)